src/constants.rs

```diff
@@ -0,0 +1,3 @@
+// dimensions of main window
+pub const WINDOW_WIDTH: f32 = 550.0;
+pub const WINDOW_HEIGHT: f32 = 360.0;
```

Comment:
Height of `341.0` looks perfect once the label and input fields are on the same line